api/src/pcapi/scripts/move_offer/move_batch_offer.py

```diff
@@ -225,19 +225,30 @@ def _update_destination_venue_to_permanent(destination_venue: offerers_models.Ve
     return False
 
 
-def _create_action_history(origin_venue_id: int, destination_venue_id: int) -> None:
+def _create_action_history(
+    origin_venue_id: int, destination_venue_id: int, destination_venue_updated_to_permanent: bool
+) -> None:
     db.session.add(
         history_models.ActionHistory(
             venueId=origin_venue_id,
             actionType=history_models.ActionType.VENUE_REGULARIZATION,
             extraData={"destination_venue_id": destination_venue_id},
         )
     )
+    db.session.add(
+        history_models.ActionHistory(
+            venueId=origin_venue_id,
+            actionType=history_models.ActionType.VENUE_SOFT_DELETED,
+        )
+    )
     db.session.add(
         history_models.ActionHistory(
             venueId=destination_venue_id,
             actionType=history_models.ActionType.VENUE_REGULARIZATION,
-            extraData={"origin_venue_id": origin_venue_id},
+            extraData={
+                "origin_venue_id": origin_venue_id,
+                "updated_to_permanent": destination_venue_updated_to_permanent,
```

Comment:
Suite à ma première remarque, ce serait ici (non testé, donc à vérifier) : 
```suggestion
                "modified_info": {"isPermanent": {"new_info": True}},
```